pages/_app.tsx

```diff
@@ -1,23 +1,25 @@
 import { useEffect, useState, createContext } from "react";
-import "@/styles/globals.scss";
-import colors from "@/utils/colors";
-import Header from "@/components/Header";
+import "../styles/globals.scss";
+import colors from "../utils/colors";
+import Header from "../components/Header";
+import type { AppProps } from 'next/app'
 
-export const ThemeContext = createContext(null);
+type Themes= "bright" | "dark";
+export const ThemeContext = createContext({});
```

Comment:
give the context a type, this is for ur own good >u<

Reply:
also why switch from null to empty object? explain

Reply:
noted, not empty anymore